website/docs/index.html.markdown

```diff
@@ -51,9 +51,51 @@ data "vcfa_tm_version" "version" {
   condition         = ">= 10.7.0"
   fail_if_not_match = false
 }
+```
+
+## Example usage (API token)
+
+```hcl
+provider "vcfa" {
+  api_token            = var.api_token
+  auth_type            = "api_token"
+  org                  = "System"
+  url                  = var.vcfa_url
+  allow_unverified_ssl = var.vcfa_allow_unverified_ssl
+  logging              = true # Enables logging
+  logging_file         = "vcfa.log"
+}
+
+# Fetch the Tenant Manager version
+data "vcfa_tm_version" "version" {
+  condition         = ">= 10.7.0"
+  fail_if_not_match = false
+}
+```
+
+## Example usage (API token file)
 
+```hcl
+provider "vcfa" {
+  api_token_file       = "token.json"
+  auth_type            = "api_token_file"
+  org                  = "System"
+  url                  = var.vcfa_url
+  allow_unverified_ssl = var.vcfa_allow_unverified_ssl
+  logging              = true # Enables logging
+  logging_file         = "vcfa.log"
+}
+
+# Fetch the Tenant Manager version
+data "vcfa_tm_version" "version" {
+  condition         = ">= 10.7.0"
+  fail_if_not_match = false
+}
 ```
 
+The file containing the API token needs to be readable and writable, you can check the file syntax in the
```

Comment:
Right now if you read this for the first time, it looks like the only way is to prepare the token manually. Should we also say that one can retrieve it with that resource too?

Reply:
True, changed